### PR TITLE
aws-upi 3az: ocs_ci/deployment/aws.py

### PR DESCRIPTION
**Pull request by shreekarSS**
Fixes: https://github.com/red-hat-storage/ocs-ci/issues/1117

### CODE REVIEW

**Comment by vasukulkarni** [comment]
LGTM

**Comment by shreekarSS** [comment]
> LGTM

Needs testing with both 1AZ and 3AZ upi rhel

**Comment by shreekarSS** [comment]
> My basic question is, do we already have UPI setup with 3AZ ? because you are fetching from different RHCOS workers assuming that they all are in different zones. So I was wondering would it work ?

New 3 worker nodes should have different subnet ID for 3AZs, else with single subnet code will create 3 instances in same zone. Each worker profile has a different suffix.

File: ocs_ci/deployment/aws.py [hunk 1]
```diff
@@ -426,12 +426,15 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
...
             self.add_rhel_workers()
 
-    def gather_worker_data(self):
+    def gather_worker_data(self, suffix='no0'):
         """
         Gather various info like vpc, iam role, subnet,security group,
         cluster tag from existing RHCOS workers
+
+        Args:
+            suffix (str): suffix for the worker data
```
Comment (shylesh) [inline ocs_ci/deployment/aws.py]: Can you add default value for suffix in the doc string.

**Comment by petr-balogh** [comment]
@shreekarSS is it verified? If so please remove `Need deployment testing` label and add `Verified` label. Thanks

**Comment by shreekarSS** [comment]
> @shreekarSS is it verified? If so please remove `Need deployment testing` label and add `Verified` label. Thanks

Hey Petr, 1AZ is tested, waiting for 3AZ test result. Will update once it's finished

**Comment by shreekarSS** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/2871/ --> 1AZ
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/2876/ --> 3AZ

**Pull request merged**